app/app/src/main/java/org/omecproject/up4/cli/FlowsStatsCommand.java

```diff
@@ -0,0 +1,245 @@
+/*
+ SPDX-License-Identifier: Apache-2.0
+ SPDX-FileCopyrightText: 2022-present Open Networking Foundation <info@opennetworking.org>
+ */
+package org.omecproject.up4.cli;
+
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.Option;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.omecproject.up4.impl.DownlinkUpfFlow;
+import org.omecproject.up4.impl.Up4AdminService;
+import org.omecproject.up4.impl.UplinkUpfFlow;
+import org.onlab.packet.Ip4Address;
+import org.onosproject.cli.AbstractShellCommand;
+import org.onosproject.net.behaviour.upf.UpfCounter;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * UP4 UE traffic statistics.
+ */
+@Service
+@Command(scope = "up4", name = "upf-stats",
+        description = "Read all UE flows stats")
+public class FlowsStatsCommand extends AbstractShellCommand {
+
+    private static final String SEPARATOR = "-".repeat(40);
+    private static final float DEFAULT_SLEEP_TIME = 2; // 2 seconds
+
+    @Option(name = "-t", aliases = "--time",
+            description = "Time between counter reads (seconds)",
+            valueToShowInHelp = "2")
+    float sleepTimeS = DEFAULT_SLEEP_TIME;
+
+    @Option(name = "-s", aliases = "--session",
+            description = "UE Session ID (IPv4 UE address)",
+            valueToShowInHelp = "10.0.0.1")
+    String ueSessionId = null;
+
+    @Option(name = "-c", aliases = "--continue",
+            description = "Continue to print stats, until ctrl+c")
+    boolean cont = false;
+
+    @Option(name = "-d", aliases = "--debug",
+            description = "Print debug stats")
+    boolean debug = false;
+
+    @Override
+    protected void doExecute() throws Exception {
+        Up4AdminService adminService = get(Up4AdminService.class);
+        Ip4Address ueAddr = ueSessionId != null ? Ip4Address.valueOf(ueSessionId) : null;
```

Comment:
(Minor) Is it possible that we have IPv6 UE address in the future? Should we use a more generic `IpAddress` here?